source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1054,6 +1054,15 @@ export class CardElementPeer extends DesignerPeer {
             }
         }
 
+        if (getExcludedProperties.indexOf("isVisible") < 0) {
+            let isVisible = addLabelAndInput(card, "Initially visible:", Adaptive.ToggleInput);
+            isVisible.input.defaultValue = String(this.cardElement.isVisible);
+            isVisible.input.onValueChanged = () => {
+                this.cardElement.isVisible = isVisible.input.value == "true";
```

Comment:
should be `===` (we probably need to update those in lots of places)

Reply:
There are just so many occurrences of this in the code... I'll eventually fix them all in one shot.

Reply:
probablu file a bug/issue to track it :)